### ISSUE TITLE
arch in MSBuild() helper is mapped to x86 instead of Win32

### ISSUE THREAD

**Issue by danimtb** [conan-io/conan#2536]
To help us debug your issue please explain:

conan 1.1.0 version

In some VS solutions (*.sln*), architecture for 32 bits is defined as ``Win32`` instead of ``x86`` and ``MSBuild()`` helper fails to configure the platform:

```
$> msbuild D:/.conan\pdxd_4sv\1\source_subfolder\TAO\TAO_ACE.sln /p:Configuration=Debug /p:Platform="x86" /m:32
error MSB4126: The specified solution configuration "Debug|x86" is invalid
```

This could be workarounded using something like ``tools.replace_in_file("TAO_ACE.sln", "Win32", "x86")`` before building.

However, the ``visual_studio_multi`` generator (maybe ``visual_studio`` one too) is creating *.prop* file with ``Win32`` as arch and may lead to inconsistency.

- [x] I've read the [CONTRIBUTING guide](https://raw.githubusercontent.com/conan-io/conan/develop/.github/CONTRIBUTING.md).
- [x] I've specified the Conan version, operating system version and any tool that can be relevant.
- [x] I've explained the steps to reproduce the error or the motivation/use case of the question/suggestion.

**Comment by danimtb** [comment]
I think this issue was closed by #2546 

Do we have to look further into this issue @memsharded?

**Comment by guban** [comment]
The `tools.replace_in_file()` workaround might result in unwanted changes, especially when it has to be applied to a single project file that has no associated solution. A better workaround is to pass `platforms={"x86":"Win32"}` to `MSbuild` tool.
